app/styles/main.scss

```diff
@@ -252,6 +251,37 @@ body {
         }
     }
 
+    .db-page-header {
+        @extend .ad-page-header;
+        .container {
+            height: inherit;;
+            .row {
+                height: inherit;
+                div {
+                    height: inherit;
+                    padding-left: 0;
+                    display: flex;
+                    justify-content:center;
+                    align-content:center;
+                    flex-direction:column;
+                    h3 {
```

Comment:
At the moment only h3 and h4 are supported. If needed other h:s should be implemented.